imagepuller/internal/service/verify.go

```diff
@@ -92,6 +91,15 @@ func (s *ImagePullerService) storeAndVerifyLayers(log *slog.Logger, remoteImg gc
 
 	previousLayer := ""
 	for idx, layer := range layers {
+		ldManifest := manifest.Layers[idx].Digest.String()
```

Comment:
`ldManifest` is a layer digest, right? In that case, `layerDigest` would be a more intuitive variable name.